app/controllers/books_controller.rb

```diff
@@ -0,0 +1,14 @@
+class BooksController < ApplicationController
+  respond_to :json
+  skip_before_action :verify_authenticity_token
+  before_action :authorize_request
+
+  def show
+    @abook = Book.find_by(id: params[:id])
```

Comment:
if you receive a params[:id] that does not exist in your table, you'd return a status code 500. You should rescue that exception and return a 404